### PR TITLE
Update delete command to display failure message for associated event/vendor

### PR DESCRIPTION
**Pull request by Jasperabez**
- Created a new exception under `model/commons/exceptions` for handling associated delete fails
   - could be made to inherit from a parent delete exception class but don't see a need to
- Update delete commands to display error msg when trying to delete an associated event/model
   - does not show which event it is associated to, just whether it is associated to any

### CODE REVIEW

**Comment by Jasperabez** [comment]
@C5hives @kevin-pek ready to be reviewed and merged

File: src/main/java/seedu/address/logic/commands/DeleteEventCommand.java [hunk 1]
```diff
@@ -24,6 +25,8 @@ public class DeleteEventCommand extends DeleteCommand {
             + "Example: " + COMMAND_WORD + " " + PREFIX_EVENT + "1";
 
     public static final String MESSAGE_DELETE_EVENT_SUCCESS = "Deleted Event: %1$s";
+    public static final String MESSAGE_DELETE_EVENT_ASSOCIATED_FAILURE =
```
Comment (C5hives) [inline src/main/java/seedu/address/logic/commands/DeleteEventCommand.java:28]: Nitpick: Would naming this constant as `MESSAGE_DELETE_EVENT_ASSOCIATION_FAILURE` make more sense?
The current name could mean an event-associated failure, which isn't what this constant represents.

File: src/main/java/seedu/address/logic/commands/DeleteVendorCommand.java [hunk 2]
```diff
@@ -24,6 +25,8 @@ public class DeleteVendorCommand extends DeleteCommand {
             + "Example: " + COMMAND_WORD + " " + PREFIX_VENDOR + "1";
 
     public static final String MESSAGE_DELETE_VENDOR_SUCCESS = "Deleted Vendor: %1$s";
+    public static final String MESSAGE_DELETE_VENDOR_ASSOCIATED_FAILURE =
```
Comment (C5hives) [inline src/main/java/seedu/address/logic/commands/DeleteVendorCommand.java:28]: Same as above comment.

**Review by C5hives — CHANGES_REQUESTED** [review]
Looks good (except for one variable naming suggestion)!

**Review by C5hives — APPROVED** [review]
Nice!

**Pull request merged**